### DISCUSSION TITLE
How to override Enter key event handler/s in HeadlessUI Combobox / ComboboxInput / etc. in Vue.js

### DISCUSSION THREAD

**Discussion by mattburnett-repo** [tailwindlabs/tailwindcss#13874] [Help]
Hi there,

Is there any way to override the default handler for Enter key events in a Vue app? The project is using Vue 3 / Nuxt.

I'm prototyping a Command Palette, following the Tailwind YouTube video https://youtu.be/-jix4KyxLuQ?si=oRbe2v4iqItyfByn, using Vue instead of React.

The state of the Command Palette is:
User has tabbed into ComboboxInput and entered search text
ComboboxOptions displays resulting searched items as ComboboxItem(s)
User up / down keys through the ComboxItems

At this point, the user can click on the ComboboxItem and go to the requested resource. Using NuxtLink on each ComboxItem for this. No problems there.

I'd like to improve the UX and allow the user to press the Enter key on any selected ComboboxItem and have the same result as clicking. However, I can't seem to get the @keypress.enter / @keydown .enter event handler to respond, no matter where I add it in the various Combobox elements.

Is this possible? How to do it, if so?

Thanks in advance!

**Comment by lessthomas** [comment]
@mattburnett-repo did you solve this?
